packages/babel-helper-plugin-test-runner/package.json

```diff
@@ -36,14 +36,18 @@
   "author": "The Babel Team (https://babel.dev/team)",
   "conditions": {
     "BABEL_8_BREAKING": [
-      null,
+      {
+        "exports": {
+          ".": {
+            "import": "./esm.mjs",
+            "default": "./lib/index.js"
+          },
+          "./package.json": "./package.json"
+        }
+      },
       {
         "exports": null
       }
     ]
-  },
-  "exports": {
-    ".": "./lib/index.js",
-    "./package.json": "./package.json"
```

Comment:
Q: Why should we move the `exports` to conditions `if-else`? I perceive that it is to use Babel 7 export-less package.json but don't know what is breaking. I expect every package will eventually come with top level `exports`, right?

Reply:
Actually, I removed this because the file already contains a complex `"exports"` that lets it work in Babel 7 while also having an ESM entry point. The exports in `BABEL_8_BREAKING` are a simpler version of what this package already has (you can see it by expanding the diff).